README.md

```diff
@@ -30,8 +30,17 @@ All the yaml in the directory and all subdirectories will be used as part of the
 ## Supported Argo CD Versions
 We aim to match the [Argo CD supported versions](https://argo-cd.readthedocs.io/en/stable/operator-manual/installation/#supported-versions) by testing against the Argo CD N and N -1 versions of Argo CD. You can see the current versions of Argo CD that we test against by looking in the [CI bootstrap directory](.github/workflows/assets/bootstrap) in this repo.
 
-# Installing as an Argo CD Plugin
-You can use [our Kustomization example](examples/installation/argocd) to install Argo CD and to bootstrap the installation of the plugin at the same time. However the steps below will detail what is required should you wish to do things more manually.
+# Installing as an Argo CD Sidecar Plugin
+You can use [our Kustomization example](examples/installation/argocd) to install Argo CD and to bootstrap the installation of the plugin at the same time.
```

Comment:
This documents ye olde nawty way, and should thus move further down the file to try to encourage the new encouraged way instead.

Reply:
I might just ditch this line altogether.

Reply:
I'd like a replacement with a sidecar way documented really.